content/docs/querying/api.md

```diff
@@ -0,0 +1,341 @@
+---
+title: HTTP API
+sort_rank: 7
+---
+
+# HTTP API
+
+The current stable HTTP API is reachable under `/api/v1` on a Prometheus
+server. Any non-breaking additions will be added under that endpoint. There are
+no minor versions of the API.
+
+## Format overview
+
+The API response format is JSON. Every successful API request returns a `2xx`
+status code.
+
+Invalid requests that reach the API handlers return a JSON error object
+and the HTTP response code `422 Unprocessable Entity`
+([RFC4918](http://tools.ietf.org/html/rfc4918#page-78)). Other non-`2xx`
+codes may be returned from outer HTTP handling layers.
+
+The JSON response envelope format is as follows:
+
+```
+{
+  "status": "success" | "error",
+  "data": <data>,
+
+  // Only set if status == "error". The data field may still hold
+  // additional data.
+  "errorType": "<string>",
+  "error": "<string>"
+}
+```
+
+Input timestamps may be provided either in
+[RFC3339](https://www.ietf.org/rfc/rfc3339.txt) format or as a Unix timestamp
+in seconds, with optional decimal places for sub-second precision. Output
+timestamps are always represented as Unix timestamps in seconds.
+
+Names of query parameters that may be repeated end with `[]`.
+
+`<series_selector>` placeholders refer to Prometheus [time series
+selectors](/docs/querying/basics/#time-series-selectors) like
+`http_requests_total` or `http_requests_total{method=~"^GET|POST$"}` and need
+to be URL-encoded.
+
+`<duration>` placeholders refer to Prometheus duration strings of the form
+`[0-9]+[smhdwy]`. For example, `5m` refers to a duration of 5 minutes.
```

Comment:
Because they are hardly used and everyone has a very clear perception of what `30d`, `90d`, or `360d` are.
Days have leap seconds - years have whole days...

Meh - we can keep them - still think days are superior in any scenario.